src/components/cart.lineitem.jsx

```diff
@@ -30,6 +30,9 @@ import {
 import imgPlaceholder from '../images/img-placeholder.png';
 import cortexFetch from '../utils/Cortex';
 import './cart.lineitem.less';
+import epConfig from '../ep.config.json';
```

Comment:
This is not necessary as it's already loaded below:
`const Config = require('Config');`

Reply:
Removed